### PR TITLE
LPD-28905 Improve method

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:playwright

**Comment by victorg1991** [comment]
ci:stop

**Comment by victorg1991** [comment]
ci:test:playwright

File: modules/test/playwright/tests/layout-content-page-editor-web/mapping.spec.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	await clickAndExpectToBeHidden({
+		target: page.locator('.modal-dialog'),
+		trigger: articleCard,
+	});
+
+	await page
+		.getByLabel('Field')
+		.selectOption({value: 'DDMStructure_Country'});
+
+	const fragment = page.locator('.component-heading');
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-content-page-editor-web/mapping.spec.ts:81]: Maybe I would add some more comment from here to the end of the test so the steps we are doing are more clear, what do you think? 😛

**Review by sandrodw3 — COMMENTED** [review]
Weeee LGTM! 🥰 I just suggested some simplifications, don't know if it's possible. If not, just forward 🚀

File: modules/test/playwright/tests/layout-content-page-editor-web/mapping.spec.ts (hunk 1 above)
Comment (victorg1991) [reply modules/test/playwright/tests/layout-content-page-editor-web/mapping.spec.ts:81]: Sure I'll update that :)

File: modules/test/playwright/tests/layout-content-page-editor-web/mapping.spec.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+				key: 'BASIC_COMPONENT-heading',
+			}),
+		]),
+		siteId: wemSite.id,
+		title: 'Test Page Name',
+	});
+
+	await pageEditorPage.goto(layout, wemSite.friendlyUrlPath);
+
+	// Map editable to repeatable field Country
```
[the remark this answers is not in the document]
Comment (victorg1991) [reply modules/test/playwright/tests/layout-content-page-editor-web/mapping.spec.ts:56]: Yeah I tried and I can't do it. I rather postpone this change, I rather have this pr in without any more complications :joy: but I promise that I'll change it :D

**Comment by victorg1991** [comment]
ci:test:playwright

**Comment by victorg1991** [comment]
ci:test:playwright

**Comment by victorg1991** [comment]
ci:test:playwright

**Pull request closed without merging**